src/geometry.h

```diff
@@ -0,0 +1,1104 @@
+#pragma once
+
+#include "turner.h"
+
+#include <algorithm>
+#include <array>
+#include <cassert>
+#include <cmath>
+#include <string>
+
+namespace turner {
```

Comment:
We could do it, however geometry is so fundamental, that you definitely don't want to put namespace name in front of each usage. Normally, you have namespaces to avoid clashes, not to package something together. It is not Java.